gokart/mypy.py

```diff
@@ -404,9 +406,13 @@ def is_parameter_call(expr: Expression) -> bool:
         type_info = callee.node
         if type_info is None and isinstance(callee.expr, NameExpr):
             return PARAMETER_FULLNAME_MATCHER.match(f'{callee.expr.name}.{callee.name}') is not None
-        if isinstance(type_info, TypeInfo) and PARAMETER_FULLNAME_MATCHER.match(type_info.fullname):
-            return True
+    elif isinstance(callee, NameExpr):
```

Comment:
These conditions are very complicated to review, because these lines are determined by print debugging by tests.
So please check the test cases for this update